build/common/installer/scripts/tomlparser-prom-customconfig.rb

```diff
@@ -109,6 +113,12 @@ def createPrometheusPluginsWithNamespaceSetting(monitorKubernetesPods, monitorKu
     new_contents = new_contents.gsub("$AZMON_TELEGRAF_CUSTOM_PROM_KUBERNETES_FIELD_SELECTOR", "# Commenting this out since new plugins will be created per namespace\n  # $AZMON_TELEGRAF_CUSTOM_PROM_KUBERNETES_FIELD_SELECTOR")
     new_contents = new_contents.gsub("$AZMON_TELEGRAF_CUSTOM_PROM_SCRAPE_SCOPE", "# Commenting this out since new plugins will be created per namespace\n  # $AZMON_TELEGRAF_CUSTOM_PROM_SCRAPE_SCOPE")
 
+    timeout_config_key = "timeout"
+    if is_windows?
+      # For windows, the timeout config key is different because of old version of telegraf
```

Comment:
sorry I forgot why are using different versions of telegraf for linux and windows?

Reply:
> sorry I forgot why are using different versions of telegraf for linux and windows?

> the version which we are using in Linux doesnt work on Windows.  Because of this, I have to revert the telegraf version for the windows.

Reply:
when we update telegraf on windows next we should remember to update here as well. maybe we should add a note in setup.ps1 next to telegraf installation>?

Reply:
> when we update telegraf on windows next we should remember to update here as well. maybe we should add a note in setup.ps1 next to telegraf installation>?

done.